src/main/java/com/amazon/dlic/auth/http/saml/HTTPSamlAuthenticator.java

```diff
@@ -338,12 +338,12 @@ private Settings createJwtAuthenticatorSettings(Settings settings) {
 
         settingsBuilder.put(jwtSettings);
 
-        if (jwtSettings.get("roles_key") == null && settings.get("roles_key") != null) {
-            settingsBuilder.put("roles_key", "roles");
+        if (jwtSettings.get("roles_key") == null) {
+            settingsBuilder.put("roles_key", settings.get("roles_key") !=null ? settings.get("roles_key") : "roles");
```

Comment:
You can just do `settings.get("roles_key", "roles")` which puts default `roles` is `roles_key` is absent

Reply:
Thanks, will do